src/macro/eval/evalStdLib.ml

```diff
@@ -3725,4 +3755,25 @@ let init_standard_library builtins =
 		"addChar",StdUtf8.addChar;
 		"toString",StdUtf8.toString;
 	];
+	init_fields builtins (["eval";"_NativeString"],"NativeString_Impl_") [
+		"fromBytes",StdNativeString.from_bytes;
+		"fromString",StdNativeString.from_string;
+		"toBytes",StdNativeString.to_bytes;
+		"toString",StdNativeString.to_string;
+		"equals",StdNativeString.equals;
+	] [];
+	init_fields builtins (["eval"],"Unix") [
```

Comment:
`Unix` or `Posix`?

Reply:
Initially I was thinking about a separate module for OCaml's `Unix` module. Hence the naming.
But it probably doesn't matter now, because I remembered you already did some work towards integrating libuv into eval. That looks like a better option than `Unix` module.